Doc/library/socket.rst

```diff
@@ -348,7 +348,7 @@ Constants
           AF_INET6
 
    These constants represent the address (and protocol) families, used for the
-   first argument to :func:`.socket`.  If the :const:`AF_UNIX` constant is not
+   first argument to :class:`.socket`.  If the :const:`AF_UNIX` constant is not
```

Comment:
"the first argument to socket()`. Functions take arguments. Classes have methods and attributes.

Reply:
If the reader clicks on that link, the target goes to https://cpython-previews--110114.org.readthedocs.build/en/110114/library/socket.html#socket.socket, which displays a callable signature. I take your point, but I think from the user's perspective, the difference is less important -- as you said, that it's a callable matters more.

Reply:
Right, and all callables are rendered with `()`. This is why role :func: is used.

It does not matter how it is declared. If it is used in the context of callable, it should be :func:.

Reply:
Ahh, I missed this initially.